extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -7,45 +7,59 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { memo } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import * as episodeIcons from '../icons/episode-icons';
+import * as trackIcons from '../icons/track-icons';
 import { STATE_ERROR, STATE_PLAYING } from '../constants';
 
 const TrackIcon = ( { isPlaying, isError, className } ) => {
-	let name;
+	let hiddenText, name;
 
 	if ( isError ) {
 		name = 'error';
+		/* translators: This is text to describe the current state. This will go before the track title, such as "Error: [The title of the track]" */
+		hiddenText = __( 'Error:', 'jetpack' );
 	} else if ( isPlaying ) {
 		name = 'playing';
+		/* translators: Text to describe the current state. This will go before the track title, such as "Playing: [The title of the track]" */
+		hiddenText = __( 'Playing:', 'jetpack' );
 	}
 
-	const icon = episodeIcons[ name ];
+	const icon = trackIcons[ name ];
 
 	if ( ! icon ) {
 		// Return empty element - we need it for layout purposes.
 		return <span className={ className } />;
 	}
 
-	return <span className={ `${ className } ${ className }--${ name }` }>{ icon }</span>;
+	return (
+		<span className={ `${ className } ${ className }--${ name }` }>
+			{ /* Intentional space left after hiddenText */ }
+			<span className="jetpack-podcast-player--visually-hidden">{ `${ hiddenText } ` }</span>
+			{ icon }
+		</span>
+	);
 };
 
 import { getColorClassName } from '../utils';
 
-const TrackError = memo( ( { link } ) => (
-	<div className="jetpack-podcast-player__episode-error">
-		{ __( 'Episode unavailable', 'jetpack' ) }{ ' ' }
+const TrackError = memo( ( { link, title } ) => (
+	<div className="jetpack-podcast-player__track-error">
+		{ __( 'Episode unavailable', 'jetpack' ) }
 		{ link && (
 			<span>
-				{ '(' }
+				{ ' - ' }
```

Comment:
This dash needs to be translated too. Can we include it in the string above and add a translation comment it will be followed by the link for opening an episode? I understand it will be colored differently but I think that's a tiny price to pay to have properly declared translations.

Reply:
I've tried to get around this by making "Episode unavailable" into a sentence ending with a period.

Reply:
This needs a different solution from the one we had and from the one I've tried introducing.

cc: @jeryj

Reply:
I'm not sure how it should be grouped to be translatable either 😞 I think a hard-stop `.` makes sense to help make it two separate items. For further context and to make sure we find a good solution, we had more conversation here: https://github.com/Automattic/jetpack/pull/15207#discussion_r401439353

@davemart-in @jeherve @akirk

Reply:
Just to recap it, we have three different things happening:
- "Episode unavailable" message
- hidden text with episode title (for screen readers)
- A link to open episode in a new tab

There have gone through several stylistic changes, having the link in parentheses, having a dash between the error message and the link. I just don't know how to solve this, because we cannot use `__experimentalCreateInterpolateElement` (which is too new in Gutenberg and not yet available everywhere where Jetpack runs).

Reply:
Why does the dash need to be translated? It's just a dash separator. There is no need for it to be changed no matter which language is used. Can we just leave it untranslated?

Reply:
As far as I understand the problematic, different locales use different ways to separate things like this and that's why colons, hyphens and similar are part of translations too.

Reply:
Can we iterate on this in a follow-up PR? It doesn't seem to be a regression